<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectGenomeWidget</class>
 <widget class="QWidget" name="SelectGenomeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>116</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Genome</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="outerFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="m_genomeSelector">
        <property name="toolTip">
         <string>genome</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="m_addButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../qosmic.qrc">
             <normaloff>:/icons/silk/bullet_add.xpm</normaloff>:/icons/silk/bullet_add.xpm</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_deleteButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../qosmic.qrc">
             <normaloff>:/icons/silk/bullet_delete.xpm</normaloff>:/icons/silk/bullet_delete.xpm</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QScrollArea" name="scrollArea">
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="m_buttonFrame">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qosmic.qrc"/>
 </resources>
 <connections/>
</ui>
